crates/aiken-lang/src/parser/token.rs

```diff
@@ -182,6 +183,7 @@ impl fmt::Display for Token {
             Token::Once => "once",
             Token::Validator => "validator",
             Token::Via => "via",
+            Token::Benchmark => "benchmark",
```

Comment:
Let's call it `bench`; not so much because it's shorter but because it's quite often used in benchmarking terminology (bench refer to the individual tests of a benchmark, which aggregates all benches).